src/Http/Middleware/FlarumPromotionHeader.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Http\Middleware;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface as Middleware;
+use Psr\Http\Server\RequestHandlerInterface;
+
+class FlarumPromotionHeader implements Middleware
+{
+    public static $enabled = true;
```

Comment:
I would prefer for this to inject `Config`, and read from that instead.

Reply:
I agree, a config option would be much better in my opinion as well.

Reply:
I don't think an extension should be needed to toggle this, hence my preference for config.php

Reply:
I like the idea of using config.php for this. Will refactor.